ranger-hub-server-bundle/src/main/java/io/appform/ranger/hub/server/bundle/configuration/RangerServerConfiguration.java

```diff
@@ -38,4 +41,13 @@ public class RangerServerConfiguration {
     @NotEmpty
     @Valid
     List<RangerUpstreamConfiguration> upstreams;
+
+    Set<String> excludedServices;
```

Comment:
Let's add @Valid

Reply:
It's a set of strings .. what will `@Valid` do?